external/kafka-0-10/src/test/scala/org/apache/spark/streaming/kafka010/KafkaRDDSuite.scala

```diff
@@ -22,12 +22,17 @@ import java.{ util => ju }
 import scala.collection.JavaConverters._
 import scala.util.Random
 
+import kafka.common.TopicAndPartition
```

Comment:
These are the older Kafka APIs right? this may all be correct, just making sure these are the classes that are needed in a Kafka 0.10 test?

Reply:
Right, LogCleaner hadn't yet been moved to the new apis, added a comment to that effect.
Think we're ok here because it's just being used to mock up a compacted topic, not in the actual dstream api.